tests/asgi/test_asgi_servers.py

```diff
@@ -188,7 +201,11 @@ def server_base_url(request):
 
             yield base_url
 
-        assert server.returncode == 0
+        # NOTE: In the Travis CI environment, sending CTRL+BREAK works but
+        #   results in a non-zero return code (3221225786 observed?).
```

Comment:
That number is the normal exit code after a ctrl break

Reply:
The exit code does not need to be 0 on Unix-like either.
It is normal that, for instance, on Linux , terminating a program with a signal `N` makes it return an exit code `-N`.